float32[] detection_info
